src/webapi/element.rs

```diff
@@ -42,6 +42,26 @@ pub trait IElement: IEventTarget {
             js!( return @{self.as_ref()}.querySelectorAll( @{selector} ); ).into_reference_unchecked().unwrap()
         }
     }
+    
+     /// A property which represents the inner html of a element and its descendants as a DOM string.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML)
+    fn inner_html( &self ) -> Option< String > {
+        js!(
+            return @{self.as_ref()}.innerHTML;
+        ).into_string()
+    }
+
+    /// Sets the inner html of this element; calling this removes all
+    /// of node's children and replaces them with html elements
+    /// of the given DOM string.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML)
+    fn set_inner_html( &self, text: &str ) {
```

Comment:
According to specs [this can return an exception](https://w3c.github.io/DOM-Parsing/#extensions-to-the-element-interface) when the text is not well formed HTML **and** the document is XML.

For example:

```
var doc = document.implementation.createDocument("", "", null);
var e = doc.createElement("div");
e.innerHTML = "<foo"
```

will throw:

```
SyntaxError: An invalid or illegal string was specified
XML Parsing Error: not well-formed
Location: 
Line Number 1, Column 10:
```

So we should catch the exception and turn it into a `Result`. (e.g. see `INode::remove_child`)

Please also add a test for this; you can use the JS snippet I've pasted here. (You can take a look at the issue #59 if you don't know how to run Web API tests.)